inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -58,3 +58,31 @@ $bslib-checkbox-radio-margin-right: 0.35em !default;
     }
   }
 }
+
+@if ($enable-shadows) {
+  .bslib-value-box {
+    box-shadow: $box-shadow;
+  }
+}
+
+.bslib-value-box.default .value-box-showcase > svg:is(.bi, .fa) {
+  // Requires icon-gradient.svg included as singleton in value_box()
+	fill: url('#bslib---icon-gradient') #007bc2 !important;
+}
+
+.bslib-value-box {
+  &.text-info,
+  &.text-cyan.text-cyan {
+    color: #028097 !important; // Contrast 4.63
+  }
+  &.text-light {
+    color: $gray-600 !important;
+  }
+  &.text-yellow.text-yellow,
+  &.text-warning.text-warning {
+    color: #A87600; // Contrast 4 (okay for 16px+ text)
+  }
+  &.text-teal.text-teal {
+    color: #008558; // Contrast 4.67
+  }
+}
```

Comment:
Once we have dark color mode support, we may want to add more rules scoped under `@include color-mode(dark) { }`?